src/render/direct3d12/SDL_render_d3d12.c

```diff
@@ -53,15 +60,11 @@
 #endif
 
 #ifdef __cplusplus
-#define SAFE_RELEASE(X) \
-    if (X) {            \
-        (X)->Release(); \
-        X = NULL;       \
-    }
-#define D3D_CALL(THIS, FUNC, ...)             (THIS)->FUNC(__VA_ARGS__)
-#define D3D_CALL_RET(THIS, FUNC, RETVAL, ...) *(RETVAL) = (THIS)->FUNC(__VA_ARGS__)
 #define D3D_GUID(X)                           (X)
 #else
+#define D3D_GUID(X)                   &(X)
```

Comment:
You should remove the definition down below, otherwise we'll get redefinition errors

Reply:
Indeed, thanks!